terraform/modules/gro_ingestion_service/lambdas/test/functions/deleteEvent.function.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect, describe } from "@jest/globals"
+import { EventRequest } from "../../src/models/EventRequest"
+import { DeleteEventResponse } from "../../src/models/DeleteEventResponse"
+import { handler } from "../../src/functions/deleteEvent.function"
+
+import { dbItem } from "./dbItem"
+
+import { DynamoDBClient, dynamoDbSendFn } from "../__mocks__/@aws-sdk/client-dynamodb"
+
+const db = new DynamoDBClient()
+
+describe("Unit test for delete event handler", function () {
+    test("verifies successful response", async () => {
+        dynamoDbSendFn.mockReturnValueOnce(Promise.resolve({ Item: dbItem }))
+
+        const event: EventRequest = {
```

Comment:
we're using this a few times, should this also be a const?

Reply:
Actually its different

Reply:
Here and in enrichEvent it looks to be the below, whats the difference?
```
        const event: EventRequest = {
            id: "hash1",
        }
```